src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customfooterlibs.html

```diff
@@ -35,5 +35,9 @@
      data-sly-test.hasForm="${symbol_dollar}{formstructparser.containsFormContainer}">
     <sly data-sly-test="${symbol_dollar}{hasForm}" data-sly-call="${symbol_dollar}{clientlib.js @ categories='core.forms.components.runtime.all', async=true}"/>
 </sly>
+#if ( $datalayer == "y")
```

Comment:
Please make sure that the clientlib inclusion has to be similar to this, [https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main[…]root/apps/core/fd/components/page/v1/page/customfooterlibs.html](https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/page/v1/page/customfooterlibs.html)

Reply:
its the same

Reply:
@barshat7 Please write a comment explicitly referring the original file from where we have copied the content